openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -595,6 +622,31 @@ def build_anim(project_name, asset_name):
     load_subset(project_name, board_repre, "Background")
 
 
+def build_lipsync(project_name: str, shot_name: str):
+    """Build lipsync workfile.
+
+    Args:
+        project_name (str):  Current project name from OpenPype Session.
+        shot_name (str):  Current shot name from OpenPype Session.
+    """
+
+    casting = get_kitsu_casting(project_name, shot_name)
+    assert casting, "Failed to get kitsu casting"
+
+    for actor in casting:
+        for _ in range(actor["nb_occurences"]):
+            if actor["asset_type_name"] == "Character":
+                try:
+                    load_subset(
```

Comment:
Here you'll get an issue if you're remote and subsets are not locally available. I suggest to create a `download_kitsu_casting()` function which will take from 257 to 272 plus the get casting part you've done (`get_kitsu_casting` can be enhanced to fulfill this purpose).